sdk/keyvault/azure-security-keyvault-secrets/src/private/keyvault_protocol.hpp

```diff
@@ -156,8 +151,20 @@ namespace Azure { namespace Security { namespace KeyVault { namespace _detail {
         std::vector<std::string> const& path)
     {
       auto serialContent = serializeContentFn();
+
+      return SendRequest(context, method, serialContent, factoryFn, path);
+    }
+
+    template <class T>
```

Comment:
This is part of my fear @ahsonkhan from removing KeyVault common.  Now the protocol implementation starts to be different in this package than what it is on Keys :(  

@gearama can you keep it in sync for now, until we unify the file and find out how to re-use it from both packages?

Reply:
Unresolving just to wait for @ahsonkhan  comment.

Reply:
i would need to merge this change , i've been waiting for all the previous changes and i have a backlog of PRs that just get more complicated to merge and resolve with each passing day